build.gradle

```diff
@@ -215,8 +281,31 @@ project(':par-connector') {
         includeEmptyDirs = false
     }
 
-    task startScheduler(dependsOn: [distScheduler, installJars, reinstallRPackage]) << {
-        schedProcess = ['java', '-classpath', 'dist/lib/*', 'org.ow2.proactive.scheduler.util.SchedulerStarter', '-ln', '2', '-t', '60000'].execute(null, new File(schedHome))
+    task changeSchedulerSettings(dependsOn: distScheduler) << {
+        def schediniFile = file("${rootProject.buildDir}/scheduler/config/scheduler/settings.ini")
+        def pattern = java.util.regex.Pattern.compile("pa\\.scheduler\\.task\\.fork=.*")
```

Comment:
I don't know if it is easily to do but it would be nice to retrieve property name from `PASchedulerProperties` enum. This way, if property name changes, then it is automatically reflected on this part of code (`Pattern.quote` can probably be used to escape characters).